src/test/java/com/github/webicitybrowser/spec/fetch/FetchEngineTest.java

```diff
@@ -67,17 +99,18 @@ public FetchResponse send(FetchRequest request) {
 	}
 
 	private FetchResponse mockFetchResponse() {
-		return new FetchResponse() {
-			
-		};
-	}
-
-	private FetchRequest createRequest(String method, URL url) {
-		FetchRequestBuilder fetchRequestBuilder = FetchRequestBuilder.create();
-		fetchRequestBuilder.setMethod(method);
-		fetchRequestBuilder.setUrl(url);
+		return new FetchResponse(){
+			@Override
+			public Body body() {
+				return Body.createBody(new InputStreamReader(new ByteArrayInputStream(new byte[] {1,2,3,4})),DUMMY_BODY, -1);
+			}
 
-		return fetchRequestBuilder.build();
+			@Override
+			public boolean equals(Object o) {
+				FetchResponse fr = (FetchResponse) o;
```

Comment:
`fr` is not a good variable name. Consider calling it `otherFetchResponse`

(The o is fine as that is pretty standard)